doc.php

```diff
@@ -15,19 +15,26 @@ function endsWith($haystack, $needle) {
     header("Location: $request_uri");
     exit();
   }
-  # the URL follow this format https://www.cyberbotics.com/doc/book/page?version=tagOrBranch#anchor where version and anchor are optional
+  # the URL follow this format https://www.cyberbotics.com/doc/book/page?version=tagOrBranch&tab=C#anchor where version, tab and anchor are optional
```

Comment:
Instead of `tab`, I would call this parameter "programming language", but because it's a bit long, a better name could be `api`. What do you think?

Reply:
Also, we should list here all the valid values for this new parameter, for C or Python, it is obvious, but I guess that for C++, we cannot write `tab=C++` as the `+` character is a separator... So what shall we write for C++? `cpp` or `c%43%43`?

Reply:
> Instead of tab, I would call this parameter "programming language", but because it's a bit long, a better name could be api. What do you think?

As just discussed in the office, the tab concept is much more global than just language, currently we use it only for language but nothing in the code prevents us from using for anything else.

> Also, we should list here all the valid values for this new parameter, for C or Python, it is obvious, but I guess that for C++, we cannot write tab=C++ as the + character is a separator... So what shall we write for C++? cpp or c%43%43?

I just tested locally in Chrome, Firefox and Edge and it is working fine, I will test again locally once doc.php is updated.